packages/run-protocol/src/reserve/assetReserve.js

```diff
@@ -25,6 +25,8 @@ const nonalphanumeric = /[^A-Za-z0-9]/g;
  *
  * @property {AmountKeywordRecord} allocations
  * @property {Amount<'nat'>} shortfallBalance shortfall from liquiditation that
+ * @property {Amount<'nat'>} mintedRUN total RUN minted to date
+ * @property {Amount<'nat'>} burnedRUN total RUN burned to date
```

Comment:
let's use `Fee` instead of `RUN` per https://github.com/Agoric/agoric-sdk/issues/4800

and per https://github.com/Agoric/agoric-sdk/blob/fb4b156bdd74e28af8fb93f2da3f43c1041c1cdf/packages/run-protocol/src/vaultFactory/vaultManager.js#L46 `minted` should be second. 

"total" helps convey that this is "minted _ever_" not any other time window.

```suggestion
 * @property {Amount<'nat'>} totalFeeMinted total Fee minted to date
 * @property {Amount<'nat'>} totalFeeBurned total Fee burned to date
```

Reply:
Done